### ISSUE TITLE
"Items worth" and logging of high value items not quite working

### ISSUE THREAD

**Issue by SteinerKD** [briansd9/exile-diary#107]
I got a bit of a surprise when my net worth updated and jumped over 1k Chaos.
I had picked up a Headhunter divination card worth 11 Exalts. Nowhere in the logs does it show me ever getting more than 99 Chaos on a run though and the stats showing things worth at least 1 Exalt doesn't show the card. (in the net worth list it is shown on top as 1071.84 Chaos though)
Until the net worth update I had no clue what I had on my hand.

**Comment by briansd9** [comment]
Do you recall which map you picked it up in?

If not, maybe it was from a stacked deck you opened in your hideout?

**Comment by SteinerKD** [comment]
No, it was from a ritual, think I deferred it twice before I got it. Was going to find out from what map, but as the map log doesn't show any values above the raw one exalt drops I've had I can't find it. I always use the manual end function after maps and check how much I got from the map and never even realized what I had until the net worth update.

**Comment by briansd9** [comment]
I see, can you send me your log file `%appdata%\exile-diary\log.txt`? Will look into this.

**Comment by SteinerKD** [comment]
Of course.

[log.txt](https://github.com/briansd9/exile-diary/files/5939418/log.txt)

**Comment by briansd9** [comment]
It's the time your net worth jumped from 3774 chaos to 5070 right? 

Those two stash updates are timestamped 2021-02-06 11:59:09 and 2021-02-07 15:22:15, so you should be able to find it by checking which of your map runs during this time had a Doctor drop (or was it The Demon)?

Didn't find any errors getting your inventory in the log file, so it seems more likely that the item was logged but not properly assigned a value.

**Comment by SteinerKD** [comment]
Sorry for slow reply. I've been going forwards and backwards through the log and just can't find the card listed (The Demon). At this point I'm willing to accept your suggestion it was from a stacked deck after all. The confusion comes from that I have it in the ritual system and have deferred it a few times, so I was sure that's how I finally got it. Guess that unless I've missed it or there is a glitch I will have a second copy coming sometime soon.

**Comment by SteinerKD** [comment]
Hmm, in my latest run I picked up a Dusk Needle Twilight Blade (Elder Sword) from a Ritual worth a fair chunk of chaos orbs, I can see it in the "Picked up" part where the loot filter is used, but it is not listed at all in the list showing values (and thus not contributing to the total for the run).

**Comment by briansd9** [comment]
This price list is used for Twilight Blade: https://poe.ninja/challenge/base-types?name=Twilight%20Blade

Which value were you expecting? Elder ones are listed at 3.7 c at most...

**Comment by SteinerKD** [comment]
Hmm, weird, Been a while now but I use Awakened and it gave me a different value when checking it (hence the price I listed it for) , I put it up for sale at 15c and it sold within seconds of listing, usually only happen with severely underpriced items for me. The point was that it wasn't listed at all in the list of values for picked up items, even at the 3.7c you mention shouldn't it be listed? Included a screengrab of that run.

![2021-02-13](https://user-images.githubusercontent.com/20935829/107846087-363d7a80-6de1-11eb-80d4-92f60de1426b.jpg)

**Comment by briansd9** [comment]
Ah, I see.

Equipment you pick up is priced according to the base type only, evaluating the mods on an item (as APoET does) is beyond the scope of this program.

As for why it didn't show up, perhaps you have it filtered out in the settings? (`Settings -> Filter -> Non-unique equipment less than ___ in value`)

**Comment by SteinerKD** [comment]
Hmm, good idea, I remember that setting, but no, no filters are set at all, nothing is being ignored or filtered.
The weird thing, as you can see from the screengrab, is that it does show up as picked up, just not evaluated even for a minimal value as if it was filtered.

BTW, thank you for effort and patience.

Ok, so it might just be the item level of the item? Seems to pick up pretty much everything I pick up otherwise.

**Comment by SteinerKD** [comment]
Sorry for bugging you, but on a slightly related issue. Today ona  delve run I got a raw Exalt drop. But after the run its not even showing as picked up in that run, nor did it add to the value OR show up in the list of "1 exalt or higher value drops" list in stats like all the other Exalt drops (all 5 of them).
Not sure why it glitched but I guess it is because I had to briefly return to my hideout to sell an item, then return to continue my run. I new run log started (automatic, didn't end it), but the exalt doesn't show in either of them.

**Comment by briansd9** [comment]
Can you send me your log file `%appdata%\exile-diary\log.txt` again? Will take a look.

This should only happen if there was an error getting your inventory (connection hiccup, poesessid expired)

**Comment by SteinerKD** [comment]
[log.txt](https://github.com/briansd9/exile-diary/files/6019279/log.txt)
[log1.txt](https://github.com/briansd9/exile-diary/files/6019280/log1.txt)

**Comment by briansd9** [comment]
Hmm, are you referring to the latest run in Azurite Mine (timestamp 2021-02-22 00:47:35, one visit to Battle-Scarred Hideout in the middle)?

If so, nothing seems out of the ordinary, no errors logging all items you picked up in the mine.

**Comment by SteinerKD** [comment]
No, it should have been the one before that I think, it was a Legion event down in the mines. I'm not 100% sure if it dropped from one of the larger Legion mobs or from one of the caches (Think it was from a cache). There should also have been a Timeless Splinter picked up which doesn't show at all in the logs so I think an entire session is missing between the last and second to last one recorded.
